coc/players.py

```diff
@@ -259,6 +260,8 @@ class SearchPlayer(BasicPlayer):
         :class:`int` - The players war star count
     town_hall:
         :class:`int` - The players TH level
+    town_hall_weapon:
+        :class:`int` - The weapon level of the TH
```

Comment:
A note that this is only present in TH12+ and it will be 0 otherwise would be nice. Not sure If a 0 level or ``None`` would be better. If it defaults to None I usually do ``Optional[:class:`int`]``, too

Reply:
Would it be optional though? Since the user isn't providing the info?  Since these aren't parameters, but responses, I'm not sure optional makes sense.

Reply:
Good point 👍